guides/gatherers.md

```diff
@@ -65,4 +66,44 @@ facts:
     argument: nodelist.node
 ```
 
-For extra information refer to [trento-project/agent/../gatherers/corosyncconf_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf_test.go)
+For more information refer to [trento-project/agent/../gatherers/corosyncconf_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf_test.go)
+
+### hosts (/etc/hosts)
+
+**Name:** `hosts`
```

Comment:
See this discussion with Xabi https://github.com/trento-project/wanda/pull/89, I changed [my PR](https://github.com/trento-project/wanda/pull/88) accordingly

Reply:
yes, I just read it and immediately came back here but you where too fast :laughing: